src/Nest/Cluster/ClusterHealth/ClusterHealthResponse.cs

```diff
@@ -55,6 +68,9 @@ public class ClusterHealthResponse : ResponseBase, IClusterHealthResponse
 		[JsonProperty("status")]
 		public Health Status { get; internal set; }
 
+		[JsonProperty("task_max_waiting_in_queue_millis")]
+		public long TaskMaxWaitTimeInQueueInMillis { get; internal set; }
```

Comment:
```suggestion
		public long TaskMaxWaitTimeInQueueInMilliseconds { get; internal set; }
```

The client typically expands this out to Milliseconds